com.gitlab.JakobDev.jdAnimatedImageEditor.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: com.gitlab.JakobDev.jdAnimatedImageEditor
+runtime: org.kde.Platform
+runtime-version: '6.2'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.2'
+command: jdanimatedimageeditor
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+cleanup:
+  - /app/lib/debug

```

Comment:
```suggestion
```

debug symbols are always extracted in a separate package.
Also cleanup rules are always relative to the `FLATPAK_DEST`